src/common/create-ignorer.js

```diff
@@ -1,8 +1,6 @@
-"use strict";
-
-const path = require("path");
-const ignore = require("ignore");
-const getFileContentOrNull = require("../utils/get-file-content-or-null.js");
+import path from "path";
```

Comment:
Can we use `node` protocol in `src`?

Reply:
https://github.com/prettier/prettier/pull/11903/files#diff-e2954b558f2aa82baff0e30964490d12942e0e251c1aa56c3294de6ec67b7cf5R182, didn't dig yet.  Let's improve it later, simply run eslint will fix that.

Reply:
There are lot's of TODOs, I'll try list them after we merge all those PRs.

Reply:
Okay. Thanks